resources/analysisTools/copyNumberEstimationWorkflow/beagle_embed_haplotypes_vcf.py

```diff
@@ -0,0 +1,54 @@
+ #!/usr/bin/python
+import argparse
+import gzip
+parser = argparse.ArgumentParser()
+parser.add_argument('--hap_file', help='Input file with phased haplotypes (i.e. output from Beagle)')
+parser.add_argument('--vcf_file', help='Input .vcf file')
+parser.add_argument('--out_file', help='Out .vcf file')
+args = parser.parse_args()
+
+if args.hap_file.split('.')[-1] == 'gz':
+    hap_file = gzip.open( args.hap_file, "r" )
+else:
+    hap_file = open( args.hap_file, "r" )
+vcf_file = open( args.vcf_file, "r" )
+outfile = open( args.out_file, "w" )
+
+hap_line = hap_file.readline()
+while hap_line[0] == "#":
+    hap_line = hap_file.readline()
+
+if hap_line:
+    hap_line = hap_line.rstrip().split()
+
+for vcf_line in vcf_file:
+    
+    if vcf_line[0] != "#":
```

Comment:
dito `!vcf_line.startswith("#")`
Consider defining a constant

```python
COMMENT_CHAR = "#"
```

or function

```python
is_comment_line(line):
    line.startswith("#")
```

to make the code more self-explanatory.